src/storage-manager.coffee

```diff
@@ -67,6 +61,9 @@ class _StorageManager
     collection.on 'remove', (model) ->
       model.invalidateCache()
 
+    collection.on 'change', (model) =>
+      @_invalidateCache(model.brainstemKey) unless model.isNew()
```

Comment:
It is useful when looking at lists without changing any data.

Examples: 

- autocompleters (and searching)
- tables / lists
- Loading the same data in different places (without sharing collection references!)